railties/lib/rails/generators/rails/app/templates/test/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class ActionSystemTestCase < ActionSystemTest::Base
+  teardown do
+    take_failed_screenshot
+    Capybara.reset_sessions!
```

Comment:
Would be nice if these two calls were inside the base class, not inside the generated application class. They're not meant to be changed.